include/ua_server.h

```diff
@@ -510,8 +510,6 @@ void UA_EXPORT UA_Server_addNetworkLayer(UA_Server *server, UA_ServerNetworkLaye
 
 /** @} */
 
-#ifndef __cplusplus /* the external nodestore does not work with c++ so far */
```

Comment:
Yes it works, 
but if you must have it for other projekt's you have to make "#ifdefs UA_EXTERNAL_NAMESPACES" in ua_server_internal.h or do other #ifdef's
``` c++
struct UA_Server {
    /* ... */
#ifdefs UA_EXTERNAL_NAMESPACES
    size_t externalNamespacesSize;
    UA_ExternalNamespace *externalNamespaces;
#endif
    /* ...*/
}
```
I can compile it with and without "UA_EXTERNAL_NAMESPACES"